JSONModel/JSONModelTransformations/JSONKeyMapper.m

```diff
@@ -171,4 +171,45 @@ +(instancetype)mapperFromUpperCaseToLowerCase
 
 }
 
++ (instancetype)mapper:(JSONKeyMapper *)baseKeyMapper
+        withExceptions:(NSDictionary *)exceptions
+{
+    NSDictionary* userToModelMap = [exceptions copy];
+    NSDictionary* userToJSONMap  = [NSDictionary dictionaryWithObjects:exceptions.allKeys forKeys:exceptions.allValues];
```

Comment:
Just because the docs say that the enumeration order is undefined doesn't mean that the enumeration order is inconsistent between keys and values. Has anyone ever documented such inconsistency?

That said, happy to change it to be extra safe.